src/main/java/org/sagebionetworks/web/client/widget/entity/EntityPageTop.java

```diff
@@ -615,6 +620,13 @@ public void updateEntityBundle(EntityBundle bundle, Long version) {
         dockerChanged(bundle);
       }
     }
+    String projectId = projectHeader != null ? projectHeader.getId() : null;
+    String entityId = entity != null ? entity.getId() : null;
+    Double versionNum = currentTargetVersionNumber != null
+      ? currentTargetVersionNumber.doubleValue()
+      : null;
+    entityCitation.configure(projectId, entityId, versionNum);
```

Comment:
Seems like this could be replaced with the utility you have below?
```suggestion
    configureEntityCitationForTab(bundle, currentTargetVersionNumber);
```

Reply:
I missed this, nice catch!